day22/permutations.py

```diff
@@ -0,0 +1,20 @@
+def permutations(arr):
```

Comment:
could have replaced the whole function with

    from itertools import permutations

;)